content/posts/macvdmtool.md

```diff
@@ -3,51 +3,56 @@ title: "DFU restore Apple Silicon Macs with macvdmtool"
 date: 2021-11-04
 draft: false
 ---
+(updated 2024-01-31, new info about Intel macs)
```

Comment:
Does Hugo render this in the HTML? I want to make sure it's hidden in the actual blog post but I like the idea of a changelog ✅ But perhaps a changelog isn't necessary as we have Git history. Let me know what you think

Reply:
I am not sure, best to remove it i think